provider/inspec.rb

```diff
@@ -67,6 +67,14 @@ def generate_documentation(data)
         default_template: 'templates/inspec/doc-template.md.erb',
         out_file: File.join(docs_folder, "google_#{data[:product_name]}_#{name}.md")
       )
+      generate_gemfile(data)
+    end
+
+    def generate_gemfile(data)
```

Comment:
Let's just copy this file over using the `files:` parameter on the inspec.yaml file. It doesn't look like there's any need to have a separate function for this.